doc/01_installation.md

```diff
@@ -4,24 +4,27 @@ This document provides step-by-step instructions to prepare and launch the Kafnu
 
 ---
 
+
 ## 1. 📦 Kafka Connect Plugins Setup
 
-Kafka Connect requires several custom and third-party `.jar` files under:
+Kafka Connect now uses a custom-built Docker image with all required plugins included. You'll need to:
+
+1. Build the required JARs (next section)
+2. Place them in the correct structure under `kafka-connect-custom/plugins/`
```

Comment:
Maybe "correct structure" is not explicit enough. What about this?

```suggestion
2. Place them in under `kafka-connect-custom/plugins/` in the same directory structure shown below.
```

or some other alternative explicit enough.